src/pages/FetchRecipe/FetchRecipe.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import './fetchRecipe.scss';
+/**
+ * @description - renders the recipes
+ * @param {object} props
+ * @returns {jsx} - returns all the recipes if found
+ */
+const FetchRecipes = ({ recipes }) => (
```

Comment:
Why do you have a *component* named `FetchRecipes` and why is it under `/pages`?